server-phoenix/lib/helios_web/controllers/web_hooks/github_controller.ex

```diff
@@ -32,7 +32,7 @@ defmodule HeliosWeb.WebHooks.GithubController do
   end
 
   defp github_push(payload) do
-    if payload["ref"] == "refs/heads/master" do
+    if Enum.member?(["refs/heads/master", "refs/heads/main"], payload["ref"]) do
```

Comment:
Ah! Nice catch.

We have some example github payloads in: `app/lib/simulate/`. I took a peek in them and it looks like github sends both a `default_branch` and `master_branch` value. Not sure what the difference is but I imagine they could future proof matching any github mainline branch is named.

I'm also wondering if we should just treat every `git push` as new commits.

Reply:
Would treating every `git push` as new commits count force pushes for rebases etc inflating the numbers? Also missed the default/master branch fields, that does seem like a good idea.